test/ecto/adapters/sql_test.exs

```diff
@@ -1,30 +1,30 @@
-defmodule Ecto.Adapters.SQLTest do
```

Comment:
We can delete this file but we need to add an integration test that shows we can pass the pool options on `start_link`.